<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibrationdialog</class>
 <widget class="QDialog" name="Calibrationdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="maxVoltLabel">
         <property name="text">
          <string>MaxVoltage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="maxVolDoubSpin"/>
       </item>
       <item>
        <widget class="QLabel" name="stepVolLabel">
         <property name="text">
          <string>StepVoltage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="stepVolDoubSpin"/>
       </item>
       <item>
        <widget class="QLabel" name="currentVolLabel">
         <property name="text">
          <string>CurrentVoltage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="currentVolDoubSpin"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="autoCalibraBtn">
       <property name="text">
        <string>autoCalibration</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="calibtabWidget">
     <column>
      <property name="text">
       <string>channel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>slope</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>offset</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="loadConfigBtn">
       <property name="text">
        <string>Loadding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="resistAllSetDpbox">
       <property name="suffix">
        <string> MÎ©</string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resistAllSetBtn">
       <property name="text">
        <string>Set all resistance</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="clearChangeBtn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveChangeBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="slopeAllsetLab">
         <property name="text">
          <string>All slope</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="slopeAllsetDpbox">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="offsetAllsetLab">
         <property name="text">
          <string>All offset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="offsetAllsetDpbox">
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="slopeOffsetResetBtn">
       <property name="text">
        <string>All reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
